src/etc/lldb_commands

```diff
@@ -1,43 +1,80 @@
-type synthetic add -l lldb_lookup.synthetic_lookup -x "^(alloc::([a-z_]+::)+)String$" --category Rust
+# Forces test-compliant formatting to all other types
+type synthetic add -l lldb_lookup.synthetic_lookup -x ".*" --category Rust
```

Comment:
Just wanted to point out that LLDB 19 also added a hook for programmatic type matching: https://lldb.llvm.org/use/variable.html#callback-based-type-matching. This could help avoid attaching DefaultSyntheticProvider to types that don't need special formatting.
Type matches seem to be cached for the entire debug session, so there may be nice performance improvements as well.